client/src/Features/Platform/PlatformRegistrationFields.ts

```diff
@@ -78,6 +78,13 @@ const toolSettingsFields: field[] = [
     isRequired: true,
     isCopyable: false
   },
+  {
+    fieldLabel: 'Audience',
+    fieldName: 'audience',
+    helpText: 'The Audience provided by the LMS for this specific registration. If Blank will use the endpoint URL.',

```

Comment:
@leestott Can you please review the helpText here once?

Reply:
yes this looks fine. We will add the details of Audience field to OpenEdx, Canvas, Moodle and Blackboard as NOT being applicable so can be left blank.

Regression testing from an engineering perspective to ensure changes do not impact other LMS installations